apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -88,49 +109,62 @@ const CalendarPaneToolbar = ({
         setOpenSchedules((prev) => !prev);
     };
 
+    useEffect(() => {
+        const handleResize = () => {
+            if (paperRef.current) {
+                const fontSize = parseFloat(getComputedStyle(paperRef.current).fontSize);
+                const widthInEm = paperRef.current.clientWidth / fontSize;
+                setIsWideEnough(widthInEm > 60);
+            }
+        };
+        handleResize();
+
+        const resizeObserver = new ResizeObserver(handleResize);
+        if (paperRef.current) {
+            resizeObserver.observe(paperRef.current);
+        }
+
+        return () => {
+            window.removeEventListener('resize', handleResize);
+        };
+    }, [paperRef]);
```

Comment:
No, please just use a media query. You can import one from MUI, i.e. `import { useMediaQuery } from '@mui/material'`

Reply:
Just replaced this all with main now that the resizing is responsive I'm not going to touch it.